akka-projection-alpakka-kafka/src/main/scala/akka/projection/scaladsl/kafka/KafkaConsumer.scala

```diff
@@ -0,0 +1,24 @@
+package akka.projection.scaladsl.kafka
+
+import akka.{Done, NotUsed}
+import akka.kafka.ConsumerMessage.{Committable, CommittableMessage}
+import akka.kafka.{CommitterSettings, ConsumerSettings, Subscription}
+import akka.kafka.scaladsl.{Committer, Consumer}
+import akka.kafka.scaladsl.Consumer.Control
+import akka.stream.scaladsl.{Flow, Source}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+// Either users need to build their own at-least-once Kafka source or we provide this convenience in Alpakka
+object KafkaConsumer {
```

Comment:
Just a convenience. We don't necessarily need such a thing. We could think if it makes sense to add it to Alpakka